packages/turbo-workspace-convert/src/getWorkspaceDetails.ts

```diff
@@ -0,0 +1,26 @@
+import managers from "./managers";
+import { Project } from "./types";
+
+async function getWorkspaceDetails({
+  workspaceRoot,
+  fallback = "yarn",
```

Comment:
We should just error here instead of having a default.

If the issue is needing to force it to parse as a particular package manager which is known, but not detectable, that's a different method and we shouldn't tack it on here.